index.html

```diff
@@ -3158,7 +3158,10 @@ <h2 id="AudioDestinationNode">
           Giving a concrete example, if the audio hardware supports 8-channel
           output, then we may set <a href=
           "#widl-AudioNode-channelCount">channelCount</a> to 8, and render 8
-          channels of output.
+          channels of output. <span class="synchronous">An
+          <code>NotSupportedError</code> MUST be fired if the implementation
+          does not support changing the <a href=
+          "#widl-AudioNode-channelCount">channelCount</a></span>.
```

Comment:
Can't set the count to 1 and have it upmixed to stereo before going out?  And why is this not considered a bug in the implementation? It seems to me that you should set maxChannelCount to 2 (always). (Chrome on Linux had this issue where maxChannelCount = 2 no matter how many channels the output device actually had.  This was only very recently fixed.)